k-yudin/LearnQA_Python_API
# Pytest Integration

- Test input characters count
- Validate cookie value
- Verify headers value
- Check User Agent using parameterized approach with tuple
- Form basis for test framework
